docs/topics/intellij-idea-plugin-reviewing-issues.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-reviewing-issues_{context}"]
+= Reviewing issues
+
+You can use the {ProductShortName} plugin icons to prioritize issues based on their severity. You can  see which issues have a _Quick Fix_ automatic code replacement and which do not.
+
+The results of an analysis are displayed in a directory format, showing the _hints_ and _classifications_ for each application analyzed.
+
+A hint is a read-only snippet of code that contains a single issue that either must or should be addressed before an application can modernized or migrated. Often a suggested Quick Fix code replacement is suggested. You can accept or ignore any suggested Quick Fixes.
+
+A classification is a file that has an issue but does not have any suggested Quick Fixes. You can edit a classification.
+
+.Procedure
+
+. In the {ProjectName} view, select a run configuration directory on the left pane.
++
+The *Report* and *Results* icons are displayed.
+. Click the *Results* icon.
++
+The modules and applications of the run configuration are displayed. Below each application, the *Hints* and *Classifications* icons are displayed.
+
+. Prioritize issues based on the following icons, which are displayed next to each hint:
+
+** image:intellij-idea-mandatory.png[Mandatory] : The issue must be fixed for a successful migration.
```

Comment:
passive voice here an the next in the item.

Reply:
Fixed.

Reply:
lgtm